Update setup-azure.md (#3433)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/git/setup-azure.md

\* Note: **DeleteSubscriptions** permission might be included in **EditSubscriptions** depending on your version of Azure.

> Some of these permissions are only accessible via the Azure DevOps API or CLI, for which documentation can be found [here](https://docs.microsoft.com/en-us/azure/devops/organizations/security/namespace-reference?view=azure-devops) and [here](https://learn.microsoft.com/en-us/cli/azure/devops?view=azure-cli-latest) respectively. We’ve also detailed more information on Azure DevOps API usage below to help accelerate the set up. Alternatively, you can use the Azure DevOps UI to enable permissions, but you cannot get the least permissioned set.

Reviewer: shortened text and added links to words

```suggestion
Some of these permissions are only accessible via the [Azure DevOps API](https://docs.microsoft.com/en-us/azure/devops/organizations/security/namespace-reference?view=azure-devops) or [CLI](https://learn.microsoft.com/en-us/cli/azure/devops?view=azure-cli-latest). We’ve also detailed more information on Azure DevOps API usage below to help accelerate the setup. Alternatively, you can use the Azure DevOps UI to enable permissions, but you cannot get the least permissioned set.
```
